engine/source/public/core/engine.hpp

```diff
@@ -11,8 +12,7 @@ namespace lumina
 
     class Engine
     {
-    private:
```

Comment:
Wouldn't this just reduce readability?

Reply:
I mean removing the private: mark

Reply:
IMO, no. Since a class is private by default, you also don't write `public:` in a struct, since that is public by default.

Reply:
I guess it is just something I need to get used to